app/models/ont/plate_with_samples_factory.rb

```diff
@@ -0,0 +1,162 @@
+# frozen_string_literal: true
+
+# Ont namespace
+module Ont
+  # PlateWithSamplesFactory
+  # A factory for bulk inserting a plate and all of its dependents
+
+  # rubocop:disable Metrics/ClassLength
```

Comment:
Probably better to fix this rather than circumvent it. We should only be using rubocop disable for legacy code.